admin_guide/out_of_resource_handling.adoc

```diff
@@ -146,26 +148,9 @@ eviction signal.
 |Available memory on the node has satisfied an eviction threshold.
 |===
 
-The node continues to report node status updates at the frequency specified by
-the `node-status-update-frequency` argument, which defaults to *10s*.
-
-[[out-of-resource-oscillation-of-node-conditions]]
-=== Oscillation of Node Conditions
-
-If a node is oscillating above and below a soft eviction threshold, but not
```

Comment:
this is important.   the default eviction-pressure-transition-period is 5m0s, so if you see a node that reports MemoryPressure, it will wait 5m0s to actually report that MemoryPressure is not longer occuring.